npm/src/db/db.ts

```diff
@@ -33,8 +33,8 @@ class DB implements DatabaseDriver {
     return decrypt(res, this.encryptionKey);
   }
 
-  async getAll(namespace): Promise<unknown[]> {
-    const res = (await this.db.getAll(namespace)) as Encrypted[];
+  async getAll(namespace, offset, limit): Promise<unknown[]> {
+    const res = (await this.db.getAll(namespace, offset, limit)) as Encrypted[];
```

Comment:
@vishalboxyhq The downstream (mongo and sql) getAll expects a number. So perhaps the conversion could happen one level above here in db ...

Reply:
Also, should we handle the case where the optional offset and limit are not passed in. Maybe return all records ?